core/tests/protractor_utils/Constants.js

```diff
@@ -0,0 +1,22 @@
+// Copyright 2017 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Provides constant global variables, for use in Protractor
+ * tests.
+ */
+
+var SKILL_STATUS_UNASSIGNED = 'Unassigned';
+
+exports.SKILL_STATUS_UNASSIGNED = SKILL_STATUS_UNASSIGNED;
```

Comment:
Could we create an enum/class and export that instead? Otherwise we're going to have to list every constant twice: once to declare and once to export, which seems needlessly verbose

Reply:
done!